src/sql/workbench/contrib/dashboard/browser/widgets/insights/insightsWidget.css

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.lastUpdated {
+	font-size: 10px;
+	margin-left: 5px;
+	color: #484644;
+}
+
+.vs-dark .lastUpdated,
+.vs-hc .lastUpdated {
+	color: #8A8886;
```

Comment:
have you tested the contrast ratio? the screenshot doesn't seem contrast enough.

Reply:
Yup this is the value from the mockup. It meets the guidelines:
![image](https://user-images.githubusercontent.com/31145923/78508556-85843a80-773c-11ea-8ccd-af7ea0e24c15.png)